public/js/Lineplot.js

```diff
@@ -214,13 +201,14 @@ Lineplot.prototype.keypressUpdate = function(keyCode) {
     var self = this;
 
     if(self.activeFeature) {
-        if(keyCode === RIGHTARROW && self.activeFeature < this.featureLength) {
+        if(keyCode === RIGHTARROW &&
+            self.activeFeature < this.featureVector.length) {
```

Comment:
The array of feature vectors is 0-indexed, but in the plot they're 1-indexed.. i.e. feature 0 maps to the x=1 co-ordinate of the line plot, feature 1 maps to x=2 etc. 

I had to think about this for more than 30 seconds, so I better add a comment explaining the indexing !